packages/web-client/README.md

```diff
@@ -1,3 +1,69 @@
 # web-client
 
-This is experimental.
+The `web-client` is a standalone package that allows you to interact with the [ownCloud Infinite Scale (oCIS)](https://github.com/owncloud/ocis/) APIs via TypeScript. It also provides convenient utilities and interfaces that are useful when working with these APIs.
+
+The supported APIs are:
+
+- Graph (drives, sharing, user & group management)
+- OCS (capabilities & url signing)
+- WebDAV (file operations)
+
+## Installation
+
+Depending on your package manager, run one of the following commands:
+
+```
+$ npm install @ownclouders/web-client
+
+$ pnpm add @ownclouders/web-client
+
+$ yarn add @ownclouders/web-client
+```
+
+## Usage
+
+To utilize the `web-client`, you must instantiate it with a base URI corresponding to your oCIS deployment and an axios instance. The axios instance, or at least its headers, is being used for all requests, necessitating the inclusion of all pertinent headers such as authorization.
```

Comment:
```suggestion
To utilize the `web-client`, you must instantiate it with a base URI corresponding to your oCIS deployment and an axios instance. The axios instance, or at least its headers, is being used for all requests, necessitating the inclusion of all pertinent headers such as authorization.
```

Could you use simpler words for `necessitating` and `pertinent`? 😆

Reply:
Damn you, ChatGPT, use simpler words! 🤖